projects/crash_ping_delays.kp/knowledge.md

```diff
@@ -0,0 +1,187 @@
+---
+title: Crash Ping Submission Delays by Channel
+authors:
+- chutten
+tags:
+- crash ping
+- delay
+created_at: 2017-01-26 00:00:00
+updated_at: 2017-01-26 13:08:54.084263
+tldr: How long does it take before we get crash pings from users in each channel?
+thumbnail: images/output_20_1.png
+---
+### Crash Ping Submission Delays by Channel
+
+This is follow-up analysis to the [Main Ping Submission and Recording Delays by Channel](https://github.com/mozilla/mozilla-reports/blob/master/projects/ping_delays.kp/knowledge.md) analysis previously performed.
```

Comment:
Well, I was going to suggest you point this link to the rtmo version of this report, but it appears that rtmo is currently down.